students/bedkowska_julita/lesson_03_functions/Exponentiation.py

```diff
@@ -0,0 +1,12 @@
+base = float(input())
+expon = int(input())
+
+
+def exponentiation(a, n):
+    if n == 0:
+        return 1
+    else:
+        return a * exponentiation(a, n - 1)
```

Comment:
It looks nice but in python You can:
return 1 if n ==0 else a * exponentiation(a, n-1)